static/app/components/charts/eventsRequest.tsx

```diff
@@ -38,6 +38,7 @@ type LoadingStatus = {
    * Whether there was an error retrieving data
    */
   errored: boolean;
+  errorMessage: null | string;
```

Comment:
nit: I think you can make errorMessage optional instead of required so you don't have to set `errorMessage: null` in some places?
```suggestion
  errorMessage?: string;
```

Reply:
Done! At first I wasn't sure if I should do this or make it optional. At the time I felt like `null` makes it more intentional, but `undefined` is also clear when dealing with types. Looks nicer this way